prac_05/emails.py

```diff
@@ -0,0 +1,30 @@
+"""
+Emails
+Estimate: 30 min
+Actual:   50 min
+"""
+
+CONFIRM_NO = ["n", "no"]
+
+
+def main():
+    email_to_name = {}
+    email = input("Email: ")
+    while email != "":
+        name = extract_name(email).title()
+        confirm = input(f"Is your name {name}? (Y/n) ").lower()
+        if confirm in CONFIRM_NO:
```

Comment:
Your code assumes the user answers yes, unless they confirm no. The question asked for us to assume the answer is no, unless the user answers "y". It is easy to miss something like that, but reading the question carefully is very important.